.github/workflows/heroku.yml

```diff
@@ -0,0 +1,23 @@
+name: deploy to heroku
+on:
+  push:
+    paths-ignore:
+      - 'README.md'
+  pull_request:
+    paths-ignore:
+      - 'README.md'
```

Comment:
Given that we only allow the job to run on pushes, we could lose this bit of config!